### PR TITLE
Tag TextWrap v0.1.4

### PR DESCRIPTION
**Pull request by sjkelly**
cc @carlobaldassi 

This adds support for precompilation on 0.4.

### CODE REVIEW

File: TextWrap/versions/0.1.4/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+julia 0.2-
```
Comment (tkelman) [inline TextWrap/versions/0.1.4/requires]: Is this still being tested on 0.2? If so it should at least remove the `-` since that refers to prereleases.

**Comment by carlobaldassi** [comment]
I think it's better if I tag it myself, so that the repository is tagged too. Will do in a short while (together with ArgParse), thanks.

File: TextWrap/versions/0.1.4/requires (hunk 1 above)
Comment (carlobaldassi) [inline TextWrap/versions/0.1.4/requires]: Yes it is, and I just pushed an update to remove the trailing dash.

Comment (tkelman) [inline TextWrap/versions/0.1.4/requires]: great thanks

**Comment by sjkelly** [comment]
:) Thanks! I forgot this wouldn't tag.

**Pull request closed without merging**